styles.css

```diff
@@ -138,4 +138,44 @@ table, th, td
 {
   width: 100%;
   resize:none;
+}
+
+/*for the season alert on the event data page*/
+/* The Modal (background) */
+.modal {
+  display: none; /* Hidden by default */
+  position: fixed; /* Stay in place */
+  z-index: 1; /* Sit on top */
+  left: 0;
+  top: 0;
+  width: 100%; /* Full width */
+  height: 100%; /* Full height */
+  overflow: auto; /* Enable scroll if needed */
+  background-color: rgb(0,0,0); /* Fallback color */
+  background-color: rgba(0,0,0,0.4); /* Black w/ opacity */
+}
+
```

Comment:
not that you have to, but why not just import the w3css library? it would cut down on the amount of stuff you have to custom make or copy/paste.